src/plugin/places/ui/placesui.js

```diff
@@ -139,14 +139,50 @@ plugin.places.ui.PlacesCtrl.prototype.hasRoot = function() {
  */
 plugin.places.ui.PlacesCtrl.prototype.export = function() {
   if (this.placesRoot_) {
-    plugin.file.kml.ui.launchTreeExport(this.placesRoot_, 'Export Places');
+    var activePlaces = this.getActivePlaces(this.placesRoot_);
+
+    var options = /** @type {os.ex.ExportOptions} */ ({
```

Comment:
Since these options are not used directly by the angular directive (it looks like you're copying them from `scope.options['blah']` to `controller['blah']` in all cases), you can:
1. Modify the /src/os/ex/export.js file to have these properties in the typedef and
2. Remove the quotes around all uses, e.g. `controller['blah'] = scope.options.blah;` instead of `controller['blah'] = scope.options['blah'];` so that the code can minify.

As a side note, "fields" and "items" are properties currently required by os.ex.ExportOptions, so I'm surprised that this builds... interesting. Make sure to look at existing uses of os.ex.ExportOptions to make sure that you're reusing the typedef in a good way.